BidnyiVitaliy/HW7/task3.py

```diff
@@ -0,0 +1,20 @@
+# Write a function that calculates the number of characters
+# included in a given string
+# • input: "hello"
+# • output: {"h":1, "e":1,"l":2,"o":1}
+
+# Напишите функцию, которая вычисляет количество символов
+# включенн в данную строку
+# • input: «hello»
+# • output: {"h":1, "e":1, "l":2, "o":1}
+
```

Comment:
good job
look at this version of code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result